guibot/region.py

```diff
@@ -1198,8 +1138,7 @@ def fill_at(self, anchor, text, dx, dy,
         """
         # NOTE: handle cases of empty value no filling anything
         if not text:
-            return
-        self.click_at(anchor, dx, dy, count=mark_clicks)
+            return self.click_at(anchor, dx, dy, count=mark_clicks)
```

Comment:
I think this is involuntary change and has to be restored back to the old version

Reply:
Done